src/tools/immutable_object_cache/CacheController.cc

```diff
@@ -118,7 +118,7 @@ void CacheController::handle_request(CacheSession* session,
       bool return_dne_path = session->client_version().empty();
       int ret = m_object_cache_store->lookup_object(
         req_read_data->pool_namespace, req_read_data->pool_id,
-        req_read_data->snap_id, req_read_data->oid, return_dne_path,
+        req_read_data->snap_id, req_read_data->object_size, req_read_data->oid, return_dne_path,
```

Comment:
Nit: is this line over 80 characters? If yes, please wrap it down to the next line.